.githooks/pre-push

```diff
@@ -20,7 +20,7 @@ if [ -s go.mod ]; then
         exit 1
     fi
 
-    go test $(go list ./... | grep -v cmd | grep -v internal/services/database_test_utils) -coverprofile cover.out 
+    go test $(go list ./... | grep -vw "apid/cmd$" | grep -v internal/services/database_test_utils) -coverprofile cover.out 
```

Comment:
Before `-v cmd` excluded any directory that included cmd. This only excludes the root cmd, not cmd/manager/cmd